include/network/SendAfter.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <QJsonObject>
+
+#include "network.hpp"
+#include "serverconnection.hpp"
+#include "server.hpp"
+
+class SendAfter : public QThread {
```

Comment:
Is there a reason why this is a thread? You can just use `QTimer` I think. Sending from multiple threads to the same connection requires synchronization.

Reply:
Not to the same connection, each connection has its thread

Reply:
The main thread is still sending/receiving stuff from each connection, in parallel with this one, right?

Reply:
Problem is I have to allocate on the heap the SendAfter() and I can't free them easily.
can i maybe say desallocaate yourself when your done with this function ? 
On way of doing it would be to have the signal destroy it ... kinda boring

Reply:
Well i wrote `delete this` in my code, i can now RIP